### PR TITLE
Removes bepis Techweb and reworks tech disks in general

### PR DESCRIPTION
**Pull request by JohnFulpWillard**
## About The Pull Request

Did you know that there's 2 types of disks Science can print?
One of them creates a whole new techweb and is used to transfer data to and from techwebs (which used to be for Theft objective, but nowadays all it's used for is Admin techweb).
The other one is these disks you can find from Lavaland & for the Limbgrower which holds nodes on them directly, they don't need anything else.

These are both printable and have very similar names which could easily confuse people, especially since they are both printed and used at the same place, being the R&D room and R&D console.

This will hopefully simplify it, by removing the base Tech disks from being printable. The only one that can be printed now is for Limbgrowers, which can't be easily mixed with the other type of Disk. Outside of that, Lavaland disks are staying the same, but I've moved Bepis disks to use this, which allows us to remove Bepis techwebs being made every single time a new bepis disk is created.

Examples of it in-game
image
image
image

## Why It's Good For The Game

There's no reason why every single Bepis node should be making a whole new techweb set with experiments to complete, roundstart nodes researched, being constantly checked if it should gain research points, have a list of papers to publish, and everything else that techwebs do.

Can you guess which disk does what in this screenshot?
image

## Changelog

:cl:
balance: You can no longer print Technology data disks. You can still print Tech disks, which hold techweb information on it, just not the one that holds up to 5 nodes.
balance: ^ Because of this, there's no way to download nodes from an RD console and upload them to an Autolathe to bypass departmental restrictions, you have to go through a Techfab/Circuit imprinter for your needs.
balance: Ones that are found cannot have anything uploaded/deleted off of them either, you can only upload them to the Web.
code: Every individual Bepis disk no longer create an entire techweb
/:cl:

### CODE REVIEW

File: tools/UpdatePaths/Scripts/74147_tech_disk_repath.txt [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# moves arm-mounted abstract chainsaw to a chainsaw subtype
```
Comment (ArcaneMusic) [inline tools/UpdatePaths/Scripts/74147_tech_disk_repath.txt:1]: Rock Raising Eyebrow dot GIF

Comment (JohnFulpWillard) [reply tools/UpdatePaths/Scripts/74147_tech_disk_repath.txt:1]: me when i copy paste

**Comment by AdipemDragon** [comment]
Pretty sure what Zamoda's on about is how you can use a disk to take a design from a techweb at an R&D console and import it into autolathes, which is currently needed for things like allowing the derelict drones to print turbine parts to actually make use of the turbine machine boards that're in there.
aka like the lavaland disks but they're DIY and anything researched goes.

**Comment by jughu** [comment]
design disks let you take a design from the techweb and import it into the autolathe, for example
I have AI consoles researched, i can take the design of the ai upload from the techweb console and import it into the autolathe so i can print it without using techwebs(guess its also a kind of way to avoid lathe taxes)

**Comment by jughu** [comment]
image
 lemme attempt this
bepis disk in the middle the blue ones are design disks, the dark purplish ones are techweb disks and the red one is the lavaland one i thinkk

**Comment by anclandor** [comment]
Does this remove the ability to import tech onto autolathes?

**Comment by JohnFulpWillard** [comment]
So this does remove that feature, however we should already be checking for ``AUTOLATHE`` build_type flag when trying to add anything to it.

I don't know if it's worth keeping this system around or trying to keep autolathes and techfabs separate, this would need a cleanup of things that can be printed at the Autolathe but aren't roundstart + is not in a tech disk.

**Comment by anclandor** [comment]
> I assume this will remove that, I'll have to check in to confirm on whether it's on these disks or techwebs ones.
> 
> I should also ask if this is worth keeping, because it sounds like a bypass departmental techfabs/lathe tax

I guess it is a way to get around lathe tax, but the effort to do it is much greater than the effort to just do a single bounty and then be able to pretty much ignore the tax for the rest of the shift.   - for this method you have to load the data onto the disk, which takes some time, becuase the interface is painful at best, and then stick it in lathe, and print it.
- actually I don't think that this does remove the ability to import to autolathes.  I think that uses the autolathe design disk which can be printed in the autolathe.

**Comment by moocowswag** [comment]
> I guess it is a way to get around lathe tax, but the effort to do it is much greater than the effort to just do a single bounty and then be able to pretty much ignore the tax for the rest of the shift. - for this method you have to load the data onto the disk, which takes some time, becuase the interface is painful at best, and then stick it in lathe, and print it.
> 
> * actually I don't think that this does remove the ability to import to autolathes.  I think that uses the autolathe design disk which can be printed in the autolathe.

It did i just tried to.

**Pull request merged**